src/a-d/Adx/Adx.BufferList.cs

```diff
@@ -172,9 +172,9 @@ public void Add(IReadOnlyList<IQuote> quotes)
     }
 
     /// <inheritdoc />
-    public new void Clear()
+    public override void Clear()
     {
-        base.Clear();
```

Comment:
Isn't `base.Clear()` the more idiomatic way of doing this?  If so, fix everywhere.  Same applies to `Add`.  If those name make it more problematic, don't make the change.